config/locales/ar.yml

```diff
@@ -3325,6 +3325,8 @@ ar:
           كانت هناك مشكلة في إعداد بطاقتك في بوابة المدفوعات الخاصة بنا.
           يرجى تحديث الصفحة والمحاولة مرة أخرى ، إذا فشلت مرة أخرى ،
           يرجى الاتصال بنا للحصول على الدعم.
+    trix:
+      urlPlaceholder: "الرجاء إدخال عنوان URL لإدراجه"
```

Comment:
Sorry I just meant to update it in `en.yml`. In general we should avoid changing other locale files directly. I'm unsure if adding new keys is a problem though.

Reply:
@mkllnk can you help with a query here? 
For convenience, Cillian has copied an existing translation to a new key (the existing translation was for angular editor, and there is an equivalent in the new trix editor). Do you know if it's ok, and if it's helpful to add new keys to the locale files?
Or should we remove this before merging?

Reply:
We should remove this addition. Transifex will handle it. It recognises existing translations.